src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Perhaps header comment can be added to describe the purpose of this interface.
Recall from tutorial that interface is a behavioural contract. Perhaps `Represents a printable entity .....`.